run-tests.php

```diff
@@ -1721,6 +1731,10 @@ function run_worker()
         define($const, $value);
     }
 
+    if(isset($greeting["output_handler"]['html']['filename'])) {
```

Comment:
nit: space between `if` and `(isset`

Reply:
I will format this code properly for styling. (phpStorm does this for me).